decidim-core/app/assets/stylesheets/decidim/extras/_register_form.scss

```diff
@@ -1,9 +0,0 @@
-.lopd-text{
```

Comment:
I would use a more generic and reusable naming.

Reply:
`.lopd-text` css class has been removed in favor of `.tos-text`, it's used to show the TOS page content in the sign-up form.

https://github.com/CodiTramuntana/decidim/blob/362a1d43d2a88baef2c75eebced3db80a8ed6b62/decidim-core/app/assets/stylesheets/decidim/modules/_signup.scss#L46

![screenshot from 2018-05-22 12-41-55](https://user-images.githubusercontent.com/210216/40357798-8e2df580-5dbd-11e8-85c2-adbe25860701.png)